pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -2552,9 +2552,10 @@ protected void internalSetOffloadPolicies(AsyncResponse asyncResponse, OffloadPo
             final String path = path(POLICIES, namespaceName.toString());
             byte[] content = globalZk().getData(path, null, nodeStat);
             Policies policies = jsonMapper().readValue(content, Policies.class);
-
-            if (offloadPolicies.getManagedLedgerOffloadDeletionLagInMillis()
-                    .equals(OffloadPolicies.DEFAULT_OFFLOAD_DELETION_LAG_IN_MILLIS)) {
+            // there is possibility that `offloadPolicies.getManagedLedgerOffloadDeletionLagInMillis()` is null.
+            if (offloadPolicies.getManagedLedgerOffloadDeletionLagInMillis() == null && OffloadPolicies.DEFAULT_OFFLOAD_DELETION_LAG_IN_MILLIS == null
+                    || offloadPolicies.getManagedLedgerOffloadDeletionLagInMillis() != null
+                    && offloadPolicies.getManagedLedgerOffloadDeletionLagInMillis().equals(OffloadPolicies.DEFAULT_OFFLOAD_DELETION_LAG_IN_MILLIS)) {
```

Comment:
Looks not related to this PR right?

Reply:
the change of code is pulsar-broker/src/test/java/org/apache/pulsar/io/PulsarFunctionE2ETest.java, 
`pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java` is another problem, why the code appear here?

Reply:
is another issue 6775

Reply:
So we can close #6775 as well.